card/db/migrate_core_cards/20150202143810_import_bootstrap_layout.rb

```diff
@@ -2,16 +2,54 @@
 
 class ImportBootstrapLayout < Card::CoreMigration
   def up
-    layout = Card.fetch "Default Layout"
+    layout = Card.fetch "Default Layout"    
     if layout
-      layout.name = "Old Layout"
+      layout.name = "Classic Layout"
       layout.update_referencers = true
       layout.save!
     end
-    
-    import_json "bootstrap_layout.json"
+        
+    import_json "bootstrap_layout.json"#, :pristine=>true, :output_file=>nil
+
+#    if unmerged.empty?
+      if layout && layout.pristine? &&
+        all = Card[:all]
+        layout_rule_card = all.fetch :trait=>:layout
+        style_rule_card  = all.fetch :trait=>:style
+        if layout_rule_card.pristine? && style_rule_card.pristine?
+          layout_rule_card.update_attributes! :content=> '[[Default Layout]]'
+          style_rule_card. update_attributes! :content=> '[[classic bootstrap skin]]'
+        end
+      end
+#    else
+#      unmerged.map! do |row|
+#        puts "didn't merge #{row['name']}"
+#        row['name'] += '+alternate'
+#        row
+#      end
+#      Card.merge_list unmerged
+#    end
+
```

Comment:
at first I wanted to make it so that *any* non-pristine style/skin would not get updated, but I kept just ending up with bizarrely bad pages.  This part is still a WIP (though no worse than before).